packages/slate-editor/src/components/SearchInput/SearchInput.tsx

```diff
@@ -92,6 +97,13 @@ export function SearchInput<T = unknown>({
 
     useRootClose(rootRef, () => setOpen(false));
 
+    useEffect(() => {
+        const unlisten = EventsEditor.addEventListener(editor, 'contact-dialog-searched', () =>
+            search(query),
```

Comment:
Is this not dangerous? `search` can go stale, if the external `getSuggestions` prop changes. Rrright?

Reply:
Theoretically yes, although the `getSuggestions` reference is passed down all the way from the plugin settings so it should be stable 🤔 

Would it be enough to wrap `search` with `useFunction`?

Reply:
> Would it be enough to wrap search with useFunction?

I guess, yes.

Reply:
d6f7d6236e587304f03a80b4aa7e2b1d4607e162